src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -92,25 +92,46 @@ function createBody(appDetails: AppDetails, nodeName: string, nodeType: string,
     const selectedResource = appDetails.resourceTree.nodes.filter(
         (data) => data.name === nodeName && data.kind.toLowerCase() === nodeType,
     )[0]
-    let requestBody = {
-        appId: getAppId(
-            appDetails.clusterId,
-            appDetails.namespace,
-            appDetails.deploymentAppType === DeploymentAppType.helm && appDetails.appType === AppType.DEVTRON_APP
-                ? `${appDetails.appName}-${appDetails.environmentName}`
-                : appDetails.appName,
-        ),
-        k8sRequest: {
-            resourceIdentifier: {
-                groupVersionKind: {
-                    Group: selectedResource.group ? selectedResource.group : '',
-                    Version: selectedResource.version ? selectedResource.version : 'v1',
-                    Kind: selectedResource.kind,
+
+    var requestBody
+
+    if (appDetails.deploymentAppType == DeploymentAppType.argo_cd) {
+        requestBody = {
```

Comment:
instead of rewriting everything we can change the value of appId only based on condition, as everything looks same for both the case accept aapId